### PR TITLE
updated README Chain and Port Info

### PR DESCRIPTION
**Pull request by c0d3ster**
-Changed to port 3000 to reflect webpack config changes
-Changed default.env values to match developer portal environment variables
-Updated usage to include link to developer portal chain set up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,15 +67,15 @@ A small example is provided in the [examples](https://github.com/EOSIO/ual-react
 cd examples
 cp default.env .env
 ```
-Now that you have a ``.env`` file, you can set your chain particulars.  Note that this file will not be version-controlled, nor should it be.
+Now that you have a ``.env`` file, so you can set environment variables for your chain particulars.  Note that this file will not be version-controlled, nor should it be.
```
Comment (chris-allnutt) [inline README.md]: so doesn't read well here and isn't necessary
`Now that you have a ``.env`` file, you can set environment variables for your chain.`

Comment (ryanwiesjahn) [inline README.md]: I don't think this sentence needs the extra "so".

File: README.md [hunk 2]
```diff
@@ -67,15 +67,15 @@ A small example is provided in the [examples](https://github.com/EOSIO/ual-react
...
-RPC_PROTOCOL=https
-RPC_HOST=api.example.net
-RPC_PORT=443
+CHAIN_ID=cf057bbfb72640471fd910bcb67639c22df9f92470936cddc1ade0e2f2e7dc4f
+RPC_PROTOCOL=http
+RPC_HOST=localhost
+RPC_PORT=8888
 ```
-These values can be edited according to the particulars of your project.  They will be used as the chain data in the example app.
+These values are taken from the local chain created by following the [Developer Portal node set up instructions](https://developers.eos.io/eosio-home/docs/getting-the-software). _(Note: if this is your first time following the tutorial you will need to install the eosio binaries [here](https://developers.eos.io/eosio-home/docs/setting-up-your-environment))_  These can be edited according to the particulars of your project if you have a different chain set up.  They will be used as the chain data in the example app.
```
Comment (chris-allnutt) [inline README.md]: These can be edited according to the  ~~particulars~~ *requirements* of your

Comment (chris-allnutt) [reply README.md]: particular is a weird word

File: README.md (hunk 1 above)
Comment (c0d3ster) [reply README.md]: agreed, will update

File: README.md (hunk 2 above)
Comment (c0d3ster) [reply README.md]: also agree, thanks for the grammar/diction check

**Pull request merged**